test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -29,37 +29,33 @@ namespace Microsoft.ML.Scenarios
 
     internal sealed class TensorFlowScenariosTestsFixture : IDisposable
     {
+        public static string tempFolder;
         public static string parentWorkspacePath;
         public static string assetsPath;
-        internal void CreateParentWorkspacePathForImageClassification()
+        internal static void CreateParentWorkspacePathForImageClassification()
         {
-            string assetsRelativePath = @"assets";
-            assetsPath = GetAbsolutePath(assetsRelativePath);
-            string workspacePath = Path.Combine(assetsPath, "cached");
+            tempFolder = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
>Path.GetTempPath(), Path.GetRandomFileName() [](start = 38, length = 44)

Are these guaranteed to be shorter?

Reply:
Yes, these are guaranteed to be shorter. Path.GetRandomeFile() only produces a 12 character random filename and in combination with Path.GetTempPath() (which is typically %USERPROFILE%\AppData\Local\Temp) the overall path will be significantly shorter than what what produced earlier. 

---
In reply to: [376950593](https://github.com/dotnet/machinelearning/pull/4816#discussion_r376950593) [](ancestors = 376950593)